src/library_glfw.js

```diff
@@ -1324,11 +1328,15 @@ var LibraryGLFW = {
   },
 
   glfwIconifyWindow: function(winid) {
-    GLFW.iconifyWindow(winid);
+#if ASSERTIONS
+    warnOnce('glfwIconifyWindow is not implemented');
```

Comment:
`GLFW.iconifyWindow` and `GLFW.restoreWindow` functions do not exist in our codebase - perhaps author intended to implement them, but never did; or perhaps we lost them in a refactor or something.. Replaced with a not implemented warning.